go.mod

```diff
@@ -10,3 +10,5 @@ require (
 	github.com/urfave/cli v1.20.0
 	gopkg.in/yaml.v2 v2.2.2 // indirect
 )
+
+go 1.13
```

Comment:
Why this change? Anyway it should probably no the in this PR as it is not really related

Reply:
Sorry I didn't want to stage this file. I will remove it